src/config.cc

```diff
@@ -141,8 +141,8 @@ Config::Config() {
       {"rocksdb.level0_file_num_compaction_trigger",
        false, new IntField(&RocksDB.level0_file_num_compaction_trigger, 4, 1, 1024)},
       {"rocksdb.enable_blob_files", false, new YesNoField(&RocksDB.enable_blob_files, false)},
-      {"rocksdb.min_blob_size", false, new IntField(&RocksDB.min_blob_size, 4096, 0, INT_MAX)},
-      {"rocksdb.blob_file_size", false, new IntField(&RocksDB.blob_file_size, 128, 0, INT_MAX)},
+      {"rocksdb.min_blob_size", false, new IntField(&RocksDB.min_blob_size, 0, 0, INT_MAX)},
```

Comment:
keep old configuration
```suggestion
      {"rocksdb.min_blob_size", false, new IntField(&RocksDB.min_blob_size, 4096, 0, INT_MAX)},
```